docs/cloud/features/scheduler/dagster.md

```diff
@@ -1,3 +1,347 @@
+
+#### Response to run status
+
+To listen for Tobiko Cloud run events, you can create a [Run Status Sensor](https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors#run-status-sensors) that listens for events on the `mirror` job and triggers your custom job in response.
+
+Within your custom job, you can do anything you like as you have full access to Python and any libraries installed in your Dagster environment.
+
+To define the custom job, you can decorate a function with `@op` to implement the task logic and a `@job` to group it into a Job:
+
+```python
+@op
+def send_email():
+    import smtplib
+
+    with smtplib.SMTP("smtp.yourdomain.com") as server:
+        server.sendmail(...)
+
+@job
+def send_email_job():
+    send_email()
+```
+
+You can also fetch a reference to the Tobiko Cloud mirror job from the Definitions we created above like so:
+
+```python
+mirror_job = tobiko_cloud_definitions.get_job_def("tobiko_cloud_mirror_run_prod")
+```
+
+Or if the Tobiko Cloud definitions are in their own Code Location and are not directly accessible from your job code, you can create a [JobSelector](https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors#cross-code-location-run-status-sensors) instead:
+
+```python
+from dagster import JobSelector
+
+mirror_job = JobSelector(job_name="tobiko_cloud_mirror_run_prod")
+```
+
+With this, you can create a `@run_status_sensor` that listens to the mirror job and triggers your job when the mirror job ends:
+
+```python
+@run_status_sensor(
+    run_status=DagsterRunStatus.SUCCESS,
+    monitored_jobs=[mirror_job],
+    request_job=send_email_job
+)
+def on_tobiko_cloud_start_run(context: RunStatusSensorContext):
+    return RunRequest()
+```
+
+You can adjust the `run_status` argument to listen for different statuses, the `monitored_jobs` argument to monitor other jobs and the `request_job` argument to trigger a different job.
+
+Here's an example that triggers a Slack notification:
+
+```python
+from dagster import run_status_sensor, job, DagsterRunStatus, EnvVar, RunRequest, RunStatusSensorContext
+from dagster_slack import SlackResource
+
+mirror_job = tobiko_cloud_definitions.get_job_def("tobiko_cloud_mirror_run_prod")
+
+@op
+def slack_op(slack: SlackResource):
+    #note: see the dagster-slack docs here: https://docs.dagster.io/_apidocs/libraries/dagster-slack
+    slack.get_client().chat_postMessage(channel="#notifications", ...)
+
+@job
+def notify_slack():
+    slack_op()
+
+@run_status_sensor(
+    run_status=DagsterRunStatus.STARTED,
+    monitored_jobs=[mirror_job],
+    request_job=notify_slack
+)
+def on_tobiko_cloud_start_run(context: RunStatusSensorContext):
+    return RunRequest()
+```
+
+![Dagster run status sensor](./dagster/run_status_sensor.png)
```

Comment:
move this image up top so it's below this #### Response to run status header

similar reasoning to the top-level image. Get someone excited to create that outcome